site/app/views/submission/HomeworkView.php

```diff
@@ -407,7 +410,9 @@ private function renderSubmitBox(Gradeable $gradeable, $graded_gradeable, $versi
             'upload_message' => $this->core->getConfig()->getUploadMessage(),
             "csrf_token" => $this->core->getCsrfToken(),
             'has_overridden_grades' => $graded_gradeable ? $graded_gradeable->hasOverriddenGrades() : false,
-            'days_to_be_charged' => $days_to_be_charged
+            'days_to_be_charged' => $days_to_be_charged,
+            'max_file_size' => $max_file_size,
+            'max_post_size' => $max_post_size
```

Comment:
Just do the calls to `Utils` here inside the array definition, no reason to set a variable for this one usage.